deploy/airgap-prepare.sh

```diff
@@ -1,16 +1,15 @@
-:'
- Copyright © 2021-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
- 
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-      http://www.apache.org/licenses/LICENSE-2.0
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
-'
+# Copyright © 2021-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 #!/bin/bash -x
```

Comment:
I think the shebang has to be the first line in the file otherwise we will still see syntax issues

Reply:
Doing this because the other scripts for [example](scripts/init_opa.sh) do the same thing for copyright. Do you want to check and change the other scripts as well?

Reply:
That script is probably obsolete. The bash line should be on top like so - https://github.com/dell/karavi-authorization/blob/main/scripts/install_karavi_auth.sh

Reply:
Have moved that to top for all the scripts.